audit/src/test/resources/config/application-mysql.yml

```diff
@@ -6,8 +6,10 @@ spring:
         password: password
         hikari:
             auto-commit: false
+            data-source-properties:
+                serverTimezone: UTC
     jpa:
-        database-platform: org.hibernate.dialect.MySQLDialect
+        database-platform: org.hibernate.dialect.MySQL8Dialect
```

Comment:
Mysql query '...for update skip locked' for outboxmessages is only supported on mysql 8+